.ci/magic-modules/generate-inspec.sh

```diff
@@ -15,10 +15,6 @@ do
   bundle exec compiler -p $i -e inspec -o "build/inspec/"
 done
 
-pushd build/inspec
-rubocop -c .rubocop.yml
```

Comment:
Why are we removing this?

Reply:
I removed it in this PR because it was giving errors that was blocking generation (`CyclomaticComplexity` was too high in generated code).

More broadly, since we're not using this for auto-correction in this step, it seems like it should be moved into `inspec-unit-test` so it doesn't block generation.

Reply:
Ah cyclomatic complexity strikes again. Ok I can add it to the unit test step and modify that rubocop file

Reply:
Could you turn off those kinds of rules (CyclomaticComplexity, AbcSize, etc). I don't think they add any value.